python/paddle/tensor/manipulation.py

```diff
@@ -3879,6 +3879,11 @@ def get_attr_shape(list_shape):
                 'int64',
                 'bool',
                 'uint16',
```

Comment:
same issue of `uint16`

Reply:
Yes, there is a historical issue with function `check_variable_and_dtype`, which temporarily use `uint16` instead of `bfloat16` in its parameters. We will make unified revisions in the future. So keep `uint16` as parameter of `check_variable_and_dtype` in `python/paddle/tensor/manipulation.py` temporarily and use `bfloat16` instead of `uint16` in unittest